content/booklets/adacore-technologies-for-airborne-software/analysis.rst

```diff
@@ -0,0 +1,3180 @@
+.. include:: common_defs.rst
+
+.. index:: single: DO-178C/ED-12C; Compliance
+
+Compliance with DO-178C/ED-12C Guidance: Analysis
+=================================================
+
+Overview
+--------
+|do-178c| uses the term "requirement" to identify the expected behavior of the system, the software, or a part thereof. The
```

Comment:
A blank line between title and first paragraph would be highly appreciated, as this improves readability for the website  maintainers. There are other instances of missing blank lines in this pull request that could be addresses as well...